Tasks/Anastasia Konuyk/Task1/task1.py

```diff
@@ -0,0 +1,10 @@
+deposit = 20000
+percent = 15/100
+years = 5
+print(f'Deposit at the beginning of the period is {deposit} BYN, procent is {percent * 100}%')
+
+for month in range(years * 12):
+    income = deposit * percent / 12
+    deposit += income
```

Comment:
there is a specific formula to calculate this percentage, the cycle is not very optimal, -1 point